README.md

```diff
@@ -149,6 +154,8 @@ packages:
 ## Package Maintenance
 The Fivetran team maintaining this package **only** maintains the latest version of the package. We highly recommend you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/github/latest/) of the package and refer to the [CHANGELOG](https://github.com/fivetran/dbt_app_reporting/blob/main/CHANGELOG.md) and release notes for more information on changes across versions.
 
+## Opinionated Decisions
```

Comment:
This package includes a DECISIONLOG.md; however, it is never referenced within the README. I felt this would be a good location to reference the DECISIONLOG.

Would you be able to add that reference here.